### ISSUE TITLE
Allow client application to provide scaling factor for model

### ISSUE THREAD

**Issue by nickgaray** [opensensorhub/osh-js#526]
Currently 3D models are scaled with a fixed scaling factor in CesiumView.  This scaling factor should be provided by the PointMarkerLayer if a 3D model is specified.

**Comment by mdhsl** [comment]
The PointMarkerLayer has a `iconScale` property that we can use. Some views are implementing it such like DeckGlView:

https://github.com/opensensorhub/osh-js/blob/dev/source/core/ui/layer/PointMarkerLayer.js#L114

We can use that property and modify the CesiumView to take it into account

**Comment by alexrobin** [comment]
Yes I agree that we should use this property and clarify in the doc that it also applies to a 3D model when it is specified
